setup.py

```diff
@@ -80,16 +80,27 @@
 if has_cython:
     cmdclass['build_ext'] = build_ext
 
+runtime_lib_dirs =  os.environ['TA_LIBRARY_PATH']
```

Comment:
how about just move the original patch into the lib_talib_dirs if statement?
`        if 'TA_LIBRARY_PATH' in os.environ:
            runtime_lib_dirs = os.environ['TA_LIBRARY_PATH']
            if runtime_lib_dirs:
                runtime_lib_dirs = runtime_lib_dirs.split(os.pathsep)
                lib_talib_dirs.extend(runtime_lib_dirs)
            else:
                runtime_lib_dirs = []`